response/core/models/event.py

```diff
@@ -0,0 +1,16 @@
+import json
+
+from django.db import models
+
+
+class Event(models.Model):
+    ACTION_EVENT_TYPE = "action_event"
+    INCIDENT_EVENT_TYPE = "incident_event"
+
+    timestamp = models.DateTimeField()
+    event_type = models.CharField(max_length=50)
+    payload = models.TextField()
+
+    def save(self, *args, **kwargs):
```

Comment:
We could add a getter here that parses the payload and returns a native dict, but since events are write-only this isn't currently needed.